test/cpp/qps/driver.cc

```diff
@@ -478,7 +474,12 @@ std::unique_ptr<ScenarioResult> RunScenario(
       client_config.add_server_targets(cli_target.c_str());
     }
   }
-
+  if (qps_server_target_override.length() > 0) {
```

Comment:
Based on the title, the change is from == 1 to <=1 , so why is the code motion needed?

Reply:
I needed to move this code because the existing code only runs of the loop executes at least once. But if `num_servers` is 0 (which is going to be the case in my planned usage), then we never enter the loop.